### PR TITLE
Run3-hcx298 Correct the scripts for Geometry related to DB for HCAL

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Correct the scripts for Geometry related to DB for HCAL

#### PR validation:

Use runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalCommonData/python/hcalSimDBConstants_cff.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+from Geometry.HcalCommonData.hcalSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalSimulationConstants_cfi    import *
+from Geometry.HcalCommonData.hcalDBConstants_cff            import *
+from Geometry.HcalCommonData.caloSimulationParameters_cff   import *
+from Geometry.HcalCommonData.hcalDDDSimConstants_cfi        import *
+from Geometry.HcalCommonData.hcalDDDRecConstants_cfi        import *
```
Comment (cvuosalo) [inline Geometry/HcalCommonData/python/hcalSimDBConstants_cff.py:8]: Are the hcalDDD*Constants_cfi needed or not? In the other above files you removed them.

**Comment by bsunanda** [comment]
@cmsbuild Please test

File: Geometry/HcalCommonData/python/hcalSimDBConstants_cff.py (hunk 1 above)
Comment (bsunanda) [reply Geometry/HcalCommonData/python/hcalSimDBConstants_cff.py:8]: I think they are

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**